src/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello, World!</title>
+  </head>
+  <body>
+      Hello, world!
+  </body>
```

Comment:
Let's go on with 2-space iindentation policy. That also refers to the text inside HTML tags

```suggestion
  <body>
    Hello, world!
  </body>
```